components/raftstore/src/coprocessor/consistency_check.rs

```diff
@@ -1,5 +1,6 @@
 // Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
 
+// #[PerformanceCriticalPath]
```

Comment:
It's not.

Reply:
Actually the whole file is to serve its compute_hash function, which is called eventually by PeerMsg::ApplyRes handler, which is on the critical path.

Reply:
ok, looks that it's indeed run in background.